pkgs/by-name/de/deltachat-rpc-server/package.nix

```diff
@@ -10,10 +9,11 @@ rustPlatform.buildRustPackage {
   inherit (libdeltachat) version src cargoLock buildInputs;
 
   nativeBuildInputs = [
-    perl
     pkg-config
   ];
 
+  OPENSSL_NO_VENDOR = true;
+
```

Comment:
There’s no `openssl` here; is this really correct?

Reply:
It inherits its build inputs from another package, and those include OpenSSL.

Reply:
Oh, sorry; I’m quite sleep‐deprived…

Reply:
That other package, libdeltachat, should also set `env.OPENSSL_NO_VENDOR = true;`.